docs/whats-new/preview-5.md

```diff
@@ -0,0 +1,81 @@
+---
+title: .NET Aspire preview 5
+description: .NET Aspire preview 5 is now available and includes many improvements and new capabilities
+ms.date: 03/28/2024
+---
+
+# .NET Aspire preview 5
+
+Breaking Changes
+- The major package split, Aspire.* and Aspire.Hosting.*
+- AppHost projects now need to reference Aspire.Hosting.AppHost
+- Each resource was split into individual packages
+ - Aspire.Hosting.Redis
+ - Aspire.Hosting.PostgreSQL
+ - Aspire.Hosting.MongoDB
+ - Aspire.Hosting.RabbitMQ 
+ - etc.
+- The same applies to the Azure resources
+ - Aspire.Hosting.Azure.Redis (etc)
+- Async all the things, various callbacks in the app model APIs
+are now async. (GetConnectionStringAsync etc.)
+
+Dashboard
+- Support for authentication 
+  - OTLP
+  - UI (Frontend)
+- LOTS of performance improvements
+   - Console log virtualization
+   - Load time improvements
+   - Trace ingestion improvements
+- Run on a single port (OTLP and UI)
+- Standalone container now forces you to choose auth
+
+Templates
```

Comment:
@DamianEdwards

Reply:
The new xUnit testing template was added, right? Were there any other updates to them?

Reply:
* removed dependencies on unstable OTel packages. 
* Enabled HTTPS and made it the default
* changed health checks to only be added in development
* enabled localization in the templates (https://github.com/dotnet/aspire/pull/2858)